Mathlib/LinearAlgebra/Matrix/ToLin.lean

```diff
@@ -112,8 +112,8 @@ theorem Matrix.vecMul_injective_iff {R : Type*} [Ring R] {M : Matrix m n R} :
     ext j
     simp [vecMul, dotProduct]
 
-lemma Matrix.linearIndependent_rows_of_isUnit {R : Type*} [Ring R] {A : Matrix m m R}
-    [DecidableEq m] (ha : IsUnit A) : LinearIndependent R (fun i ↦ A i) := by
+lemma Matrix.linearIndependent_rows_of_isUnit {R : Type*} [CommRing R] {A : Matrix m m R}
```

Comment:
Why have we lost generality here?

Reply:
I think this was an artefact of merging master. Good catch!